src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -564,26 +534,43 @@ private AIState decorationStep()
         }
 
         //We need to deal with materials
-        if (!Configurations.builderInfiniteResources && !handleMaterials(block, metadata))
+        if (!Configurations.builderInfiniteResources && !handleMaterials(block, blockState))
         {
             return this.getState();
         }
 
-        Item item = Item.getItemFromBlock(block);
-        worker.setCurrentItemOrArmor(0, item != null ? new ItemStack(item, 1) : null);
+        placeBlockAt(block, blockState, coords);
 
-        if (placeBlock(new BlockPos(x, y, z), block, metadata))
+        return findNextBlockNonSolid();
+    }
+
+    private void placeBlockAt(Block block, IBlockState blockState, BlockPos coords) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)